optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/OptaPlannerProcessor.java

```diff
@@ -298,10 +297,9 @@ private void generateConstraintVerifier(SolverConfig solverConfig,
             final List<Class<?>> planningEntityClasses = solverConfig.getEntityClassList();
             // TODO Don't duplicate defaults by using ConstraintVerifier.create(solverConfig) instead
             final ConstraintStreamImplType constraintStreamImplType =
-                    Objects.requireNonNullElse(solverConfig.getScoreDirectorFactoryConfig().getConstraintStreamImplType(),
-                            ConstraintStreamImplType.DROOLS);
-            final boolean droolsAlphaNetworkCompilationEnabled =
-                    solverConfig.getScoreDirectorFactoryConfig().isDroolsAlphaNetworkCompilationEnabled();
+                    solverConfig.getScoreDirectorFactoryConfig().getConstraintStreamImplType();
+            Boolean droolsAlphaNetworkCompilationEnabled =
+                    solverConfig.getScoreDirectorFactoryConfig().getDroolsAlphaNetworkCompilationEnabled();
```

Comment:
@Christopher-Chianelli Fixing the TODO would be the best thing to do. Unfortunately, I couldn't figure out how to pass solver config to the generated code, so I do the next best thing - I do not pass the values unless the user decided to override them.

This way, we do not duplicate the defaults. Whatever was the default stays the default in generated code.